### PR TITLE
Feature/welcome

### PR DESCRIPTION
**Pull request by ameliaeiger**
<h1>PULL REQUEST</h1>

<ul>
  <li>What does it change?</li>
  Add Welcome screen
</ul>

<h2>Type of Change</h2>

_Delete options that are not relevant_

- [ ] **Bug Fix**
- [ x] **New Feature**
- [ ] **New Change**

<h2>How has this been tested?</h2>

- [ x] **JS**
- [ x] **Dev Tools**
- [ x] **Website**

<h2>Checklist</h2>

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new bugs
- [x ] I have tagged project partners for review & it has been reviewed 

<h2>Does this PR close any issues?</h2>

### CODE REVIEW

**Review by Neil-B-Linton — APPROVED** [review]
Awesome job with the homepage feature, everything looks really good

File: App.js [hunk 1]
```diff
@@ -65,6 +67,22 @@ export default function App() {
...
+        style={styles.button}
+        icon="camera"
+        mode="contained"
+        onPress={event => {
+          console.log(event)
+          navigation.navigate("home")}}
+        >Get started!
+        </Button>
+      </View>
+)}
```
Comment (BlueNealis) [inline App.js:84]: Its beauty its grace its a welcome screen in your face.

File: Views/Dashboard.js [hunk 2]
```diff
@@ -15,54 +15,6 @@ import Data from "./data.js"
 
 
 export default function Dashboard({ handleChange, allImages }) {
```
Comment (BlueNealis) [inline Views/Dashboard.js:17]: Look at all those comment chickens DISSAPEARING

**Review by BlueNealis — APPROVED** [review]
Bootiful

**Review by laytonmaes — APPROVED** [review]

**Pull request merged**